tests/cases/compiler/acceptSymbolAsWeakType.ts

```diff
@@ -0,0 +1,22 @@
+// @lib: esnext
+// @target: esnext
```

Comment:
Does TypeScript allow us to use es2023 here yet?

```suggestion
// @lib: es2023
// @target: es2023
```